### DISCUSSION TITLE
Traj import

### DISCUSSION THREAD

**Discussion by smevs965** [csiro-robotics/raycloudtools#11] [Q&A] [answered]
running 
docker run -v c:/users/.../.../.../raytools_test/tv410 raytools rayimport 1.las PostProcessedTraj.ply

I keep getting the message:
reading: PostProcessedTraj.ply
Couldn't open file: PostProcessedTraj.ply

I have modified the header of the file to only include:
ply
format binary_little_endian 1.0
comment Any comment here
element vertex 0000000000<number of points>
property <float/double> x
property <float/double> y
property <float/double> z
property <float/double> time
end_header

I have tried this with both a hovermap trajectory file and a PosPac trajectory file. 

Thanks

**Comment by thomas-lowe** [comment]
Hi smevs965,
 your format is fine, the error message is simply that it couldn't open that binary file for reading. 
FYI it tries to open the traj file before the las file, so it may be that it would have failed to open the .las file too.

This is probably because it is looking in a folder that doesn't contain those files. So double check that your folder is correct. For example should these three dots "/.../" be two dots?

If the folder is correct then check that it has permission to read from that folder. I'm not well versed in docker, but check that it has access to that folder.

 Thanks

**Comment by tim-devereux** [comment, marked as answer]
@smevs965 

It seems the data directory is not mounted properly inside the container. Try running something like this - 

`docker run -v c:/users/.../.../.../raytools_test/tv410:/mnt raytools rayimport /mnt/1.las /mnt/PostProcessedTraj.ply`

Here the local directory is mounted inside the container at /mnt.
